spec/helpers/produce_items_helper_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ProduceItemsHelper, type: :helper do
+  context 'when viewing the produce items index' do
+    let!(:produce_item) { create(:produce_item) }
+    let!(:produce_item_two) { create(:produce_item) }
+
+    describe '#formatted_month_name' do
```

Comment:
**suggestion:** we could rethink organization a bit to read a bit better. I.e:

```rb
describe '#formatted_month_name' do
  describe 'when produce item has no seasons' do
    it 'returns an empty string'
  end

  describe 'when produce item has one season' do
    it 'returns the month name'
  end

  describe 'when produce item has multiple seasons' do
    it 'returns months separated by /'
  end
end
```

I think this brings your tests closer to the app's business language, which make them easier for humans to read and understand what you're trying to achieve. WDYT?

Reply:
I like this 🙂  it is indeed way easier to read. Rubocop complained about nested groups, I used `shared_context` to get around it. It works, but I'm not sure if this is the best solution.

Reply:
Hm, tbh I kind of disagree with Rubocop's complaint, so I'd rather use my example instead of introducing a shared_context that wouldn't be really share. So, what do you think of us tweaking the rule to our liking or disabling it? :D

Reply:
disabled it on this test 🙂